geometries/brian/M82Geometry.py

```diff
@@ -1,30 +1,27 @@
+import os
 import spiceypy as sp
 
 def main():
 
-    mk = '../kernels/asperaMetaKernel.tm'
-    spk = 'NewM82.bsp'
-    pck = 'NewM82.tpc'
+    mk = os.path.join(os.path.dirname(__file__),'../kernels/mk/asperaMetaKernel.tm')
+    spk = os.path.join(os.path.dirname(__file__),'M82spk.bsp')
```

Comment:
this second kernel should be M82pck.tpc, or the MK should be asperaMetaKernelM82.tm and the second kernels not FURNSHed at all.